pkgs/development/ocaml-modules/bistro/default.nix

```diff
@@ -16,19 +16,17 @@
 
 buildDunePackage rec {
   pname = "bistro";
-  version = "unstable-2022-05-07";
+  version = "unstable-2024-05-17";
 
   duneVersion = "3";
 
```

Comment:
This can be removed. Also, a few lines below, please raise the `minimalOCamlVersion` to `4.14`.

Reply:
Done!